dcos/package.py

```diff
@@ -1340,60 +1340,66 @@ def _data(self, path):
         full_path = os.path.join(self.path, path)
         return util.read_file(full_path)
 
-    def package_versions(self):
-        """Returns all of the available package versions, most recent first.
+    def package_revisions(self):
+        """Returns all of the available package revisions, most recent first.
 
-        Note that the result does not describe versions of the package, not
-        the software described by the package.
-
-        :returns: Available versions of this package
-        :rtype: [str]
+        :returns: Available revisions of this package
+        :rtype: [int]
         """
 
         vs = [f for f in os.listdir(self.path) if not f.startswith('.')]
         vs.reverse()
         return vs
```

Comment:
It doesn't refer to a directory name.  It refers to a revision (hence the name of the function).  We just happen to be extracting it from the directory name.